pennylane/_qubit_device.py

```diff
@@ -1358,7 +1372,11 @@ def var(self, observable, shot_range=None, bin_size=None):
             return self._dot(prob, (eigvals**2)) - self._dot(prob, eigvals) ** 2
 
         # estimate the variance
-        samples = self.sample(observable, shot_range=shot_range, bin_size=bin_size)
+        # Get samples as decimal integer values if computing ev for a MeasurementValue,
+        # otherwise the returned samples would be boolean lists
```

Comment:
Is this different from this https://github.com/PennyLaneAI/pennylane/pull/4539?

Reply:
Ah Matt's PR hadn't been merged when I added this. I'll update it now. Nice catch!